lib/mayaUsd/render/vp2RenderDelegate/CMakeLists.txt

```diff
@@ -18,7 +18,21 @@ target_sources(${PROJECT_NAME}
 )
 
 set(headers
+    basisCurves.h
+    bboxGeom.h
+    debugCodes.h
+    draw_item.h
+    instancer.h
+    material.h
+    mesh.h
     proxyRenderDelegate.h
```

Comment:
So, following the conversation with @mattyjams [here](https://github.com/Autodesk/maya-usd/pull/389/files#r400421862), it really seems like the only entry in this should be proxyRenderDelegate.h, and the rest should be removed.  I suppose I can't say this definitely, since I only checked the first 4 items, but all of those are only used locally, and with relative private-header syntax, and so don't need to be promoted.  I strongly suspect all the others in this list are the same

Reply:
@elrond79 sorry but you are looking at an outdated file here. 

As I mentioned in here https://github.com/Autodesk/maya-usd/pull/388#issuecomment-607543513 I created 3 commits to address the issues and feedbacks.

I also last night sent you a link to the changes you requested regarding <> vs "" for public headers. See https://github.com/Autodesk/maya-usd/pull/388#issuecomment-608220588

Reply:
I'm starting to get concerned that we are trying to fix much more than these 5 PRs originally intended. Let's respect the work Hamed already put into this and help him bring it to the end.

Reply:
> 
> 
> @HamedSabri-adsk - I checked the latest from this PR, [de407da](https://github.com/Autodesk/maya-usd/commit/de407da17da5087b9b50068598cb8e0240e1a354). And it's still promoting a bunch of headers. I also checked the latest from [part 5](https://github.com/Autodesk/maya-usd/pull/392) - [3324c61](https://github.com/Autodesk/maya-usd/commit/3324c613c5d6217676dac3ee1b98ca82e5578c91) - and the changes are still there.
> 

@elrond79 see changes in 886119f5

Reply:
To the larger topic - to me, I see the basic problem is that larger PRs will have longer code reviews.  I think Hamed did a commendable job in splitting up this work into multiple pieces, and I personally think they could be merged in one-at-a-time, as they're ready, since they have reasonably well-defined boundaries.  (Again, props to @HamedSabri-adsk for that!)

Yes, that does mean that other PRs may have to resolve merge conflicts multiple times.  However, I personally like merging in work in multiple smaller chunks, anyway.

Also - in the future, when dealing specically with some of these style / formatting issues, I think a better approach might be to make changes to only a few files - 1-5, say - and get feedback on those first.  That way, any ambiguities / disagreements / misunderstandings regarding the style guidelines and how they should be applied can get hammered out, without having to make changes to the entire codebase along the way.  Then, once we're all seeing on the same page on that limited subset, we can try expanding the changes to either a larger subset or the whole repo.  I think that would make things less cumbersome.

Regardless - we all want to make it to the end, and I think we're nearly there.  I believe there are only a few more outstanding notes to address, and then we can move on!

Reply:
Sounds good...from my side, I'm trying to make sure we stay in the limits of what originally was intended to be done since it's easy to grow big changes. While saying that, I must admit I missed this commit https://github.com/Autodesk/maya-usd/pull/389/commits/5d11e6f1cfa3612855fa864e32cc7d1c494d50e9 and pulled the trigger too quickly.

Reply:
@HamedSabri-adsk - ah, thanks for the link to 886119f - and I see the changes are indeed properly in Part I!